atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/command/KvsMigrationCommand.java

```diff
@@ -40,6 +41,7 @@
 @Command(name = "migrate", description = "Migrate your data from one key value service to another.")
 public class KvsMigrationCommand implements Callable<Integer> {
     private static final OutputPrinter printer = new OutputPrinter(LoggerFactory.getLogger(KvsMigrationCommand.class));
+    private static final int TRANSACTION_READ_TIMEOUT_MILLIS_OVERRIDE = 8 * 60 * 60 * 1000;
```

Comment:
Why is it hard to make the verification step use multiple transactions, same as migration.

Reply:
https://github.com/palantir/atlasdb/pull/4681